gflownet/envs/tree.py

```diff
@@ -599,11 +603,11 @@ def sample_actions(
             distr_threshold = MixtureSameFamily(mix, beta_distr)
         thresholds = distr_threshold.sample()
         # Log probs
-        logprobs[mask_cont] = distr_threshold.log_prob(thresholds)
+        logprobs[is_continuous] = distr_threshold.log_prob(thresholds)
         # Build actions
         actions_cont = [(ActionType.PICK_THRESHOLD, th.item()) for th in thresholds]
         actions = []
-        for is_discrete in mask_discrete:
+        for is_discrete in is_discrete:
```

Comment:
That's not halal. Renamed in https://github.com/alexhernandezgarcia/gflownet/pull/194 (but feel free to name it something else).

Reply:
hahahaha oops :sweat_smile: